python/api/users.py

```diff
@@ -14,26 +14,36 @@ async def read_user(email: str) -> str:
     return user.name
 
 
+@router.get("/users")
+async def read_users(_: User = Security(get_current_active_user, scopes=["users:read"])) -> list[dict[str, str]]:
+    with db_session:
```

Comment:
Suggestion: use the @db_session decorator instead of the context manager. I find it easier to read as the code is less indented, but it's up to you.